tests/drivers/spi/spi_error_cases/boards/nrf54h20dk_nrf54h20_cpuapp_fast.overlay

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	spi121_default_alt: spi121_default_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 7, 2)>,
+				<NRF_PSEL(SPIM_MISO, 7, 0)>,
+				<NRF_PSEL(SPIM_MOSI, 7, 1)>;
+		};
+	};
+
+	spi121_sleep_alt: spi121_sleep_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 7, 2)>,
+				<NRF_PSEL(SPIM_MISO, 7, 0)>,
+				<NRF_PSEL(SPIM_MOSI, 7, 1)>;
+			low-power-enable;
+		};
+	};
+
+	spis131_default_alt: spis131_default_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIS_SCK, 1, 0)>,
+				<NRF_PSEL(SPIS_MISO, 1, 9)>,
+				<NRF_PSEL(SPIS_MOSI, 1, 5)>,
+				<NRF_PSEL(SPIS_CSN, 0, 11)>;
+		};
+	};
+
+	spis131_sleep_alt: spis131_sleep_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIS_SCK, 0, 1)>,
+				<NRF_PSEL(SPIS_MISO, 0, 7)>,
+				<NRF_PSEL(SPIS_MOSI, 0, 9)>,
```

Comment:
The pin assignments here should be the same as in `spis131_default_alt` right above.

Reply:
fixed